sop/supreme_octo_potato.py

```diff
@@ -28,7 +28,7 @@ def read_code(plugins):
                                stdout=subprocess.PIPE)
 
     for output in iter(process.stdout.readline, b''):
-        output_str = output.strip().decode("utf8")
+        output_str = output.strip().decode("utf8")[8:]
```

Comment:
This could be something other than "`QR-Code:`". And we should not restrict ourselves to a single type of barcode if the zbar can also scan others.

Reply:
You're absolutely right. zbar supports a plethora of barcodes, see https://sourceforge.net/p/zbar/code/ci/default/tree/zbar/symbol.c
A fix is on the way!